packages/shorebird_cli/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: shorebird_cli
-description: The Shorebird command-line allows developers to interact with various Shorebird services.
-version: 0.0.8
+description: Command-line tool to interact with Shorebird's services.
+version: 0.0.9
```

Comment:
we need to bump the version in `src/version.dart` either manually or via `build_runner`